src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.lang.reflect.Array;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Application {
+    FigureSupplier figureSupplier = new FigureSupplier();
+    public static void main(String[] qwe) {
+        Figure [] figure = {new Square(),new Rectangle(),new RightTriangle(),
+                new IsoscelesTrapezoid()};
+        for (Figure figures : figure) {
+            System.out.println(figures);
+        }
```

Comment:
Use the 'figureSupplier' to generate random figures instead of iterating through a predefined set of figures.